test/export/StructureDefinitionExporter.test.ts

```diff
@@ -400,12 +400,12 @@ describe('StructureDefinitionExporter', () => {
     const constrainedValue = sd.findElement('Observation.value[x]');
 
     expect(baseValue.type).toHaveLength(11);
-    expect(baseValue.type[0]).toEqual({ code: 'Quantity' });
-    expect(baseValue.type[1]).toEqual({ code: 'CodeableConcept' });
-    expect(baseValue.type[2]).toEqual({ code: 'string' });
+    expect(baseValue.type[0]).toEqual({ _code: 'Quantity' });
```

Comment:
Ugh.  I hadn't thought about how this would affect the tests.  Did you consider whether it might be better to add a new function in `asserts.ts` instead?  Then this might be:
```ts
assertType(baseValue.type[0], 'Quantity');
```
or if you made the second argument `ElementDefinitionTypeJSON`, then:
```ts
assertType(baseValue.type[0], { code: 'Quantity' });
```
Or don't add an assert and instead use the class itself:
```ts
expect(baseValue.type[0]).toEqual(new ElementDefinitionType('Quantity'));
```
I don't know if any of those are _better_, but I think it's worth discussing, as making the tests knowledgeable about that _private_ field makes me feel weird -- and means we need to change a ton of tests (again) if we decide that a different approach is needed in the future.